app/controllers/twilio_webhooks_controller.rb

```diff
@@ -7,6 +7,15 @@ def update_outgoing_text_message
     head :ok
   end
 
+  def update_outbound_call
+    call = OutboundCall.find_by(twilio_sid: params["CallSid"])
```

Comment:
Since we're generating these URLs dynamically anyway, we can pass the database ID in the URL, similar to `update_outgoing_text_message`[